src/main/kotlin/racingcar/Car.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar
+
+class Car(initialMoveCount: Int) {
```

Comment:
```suggestion
class Car(initialMoveCount: Int = 0) {
```

코틀린에는 자바와 다르게 defaultValue 라는 기능이 있어요! defaultValue는 여러 생성자를 만들지 않아도 되는 장점을 가지고 있어요. 그 외도 다른 장점이 있는데요~ 미션을 하면서 천천히 알아가죠!